pytrends/request.py

```diff
@@ -390,7 +389,7 @@ def related_queries(self):
                 # in case no rising queries are found, the lines above will throw a KeyError
                 rising_df = None
 
-            result_dict[kw] = {'top': top_df, 'rising': rising_df}
+            result_dict = {'top': top_df, 'rising': rising_df}
```

Comment:
It looks like you're fundamentally changing the form of what gets returned, even for conventional keyword-bearing use-cases.